educational-webapp/src/main/webapp/alphabetGame.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Bungee Shade' rel='stylesheet'>
+    <title>Alphabet Soup</title>
+    <style>
+        body {
```

Comment:
It's best practice to move the CSS to a .css file so that people don't get confused about where to find the styling later on.

One option if you don't want to interfere with "style.css" is to make a brand new css file for just the alphabet game if you only need these CSS classes for that. Then you can add another stylesheet reference like on line 6.